dashboard/app/controllers/api/v1/pd/application/facilitator_applications_controller.rb

```diff
@@ -11,6 +11,12 @@ def new_form
     protected
 
     def on_successful_create
+      workshop = @application.find_default_workshop
+      @application.pd_workshop_id = workshop.id if workshop
+
+      fit_workshop = @application.find_default_fit_workshop
+      @application.fit_workshop_id = fit_workshop.id if fit_workshop
```

Comment:
facilitator applications can have both regular workshops and fit_workshops?

Reply:
Yep! According to Tanya:

* G1 facilitators always assigned Phoenix FiT and same local summer workshop as teachers in their region
* G2 facilitators always assigned Atlanta FiT and same local summer workshop as teachers in their region
* G3 facilitators should be assigned to same FiT & TC as teachers in their region